service/src/main/java/bio/terra/workspace/app/configuration/external/FeatureConfiguration.java

```diff
@@ -23,19 +24,33 @@ public void setAzureEnabled(boolean azureEnabled) {
     this.azureEnabled = azureEnabled;
   }
 
+  public boolean isAlpha1Enabled() {
```

Comment:
I hadn't gotten that far :)  I figured that I could use `alpha1` for going back and forth with user interface on a few endpoints before making them `v1`.

I do have a notion that we will want to revamp the entire workspace REST API and move it to v2. For that, we might want something like v2alpha1, v2alpha2, ... v2beta, v2, since we would be wanting to work out interface bugs for all of the API consumers and get some real usage before nailing it down. But I'm not really sure how we would orchestrate that process across Broad-Verily-Microsoft